client/js/array-util.js

```diff
@@ -1,12 +1,24 @@
+//Returns array from fromNum to toNum that is sequential in order, both increasing and decreasing
 const getRange = function(fromNum, toNum) {
```

Comment:
Sounds good, I changed the indentation, hopefully it will reflect this. Thank you for your assistance, please let me know if there is anything else I can do.